__tests__/src/test/java/org/example/LoyaltiesTest.java

```diff
@@ -0,0 +1,120 @@
+    public static ApiClient defaultClient = null;
+    public static LoyaltiesApi loyalties = null;
+    public static VouchersApi vouchers = null;
+
+    @BeforeAll
+    public static void beforeAll() {
+        defaultClient = Utils.getClient();
+        loyalties = new LoyaltiesApi(defaultClient);
+        vouchers = new VouchersApi(defaultClient);
+    }
+
+    @Test
+    public void updateLoyaltyCardBalanceTest() {
+        try {
+            LoyaltiesMembersBalanceUpdateRequestBody loyaltiesMembersBalanceUpdateRequestBody = new LoyaltiesMembersBalanceUpdateRequestBody();
+            loyaltiesMembersBalanceUpdateRequestBody.setPoints(1000);
+
+            LoyaltiesMembersBalanceUpdateResponseBody responseBody = loyalties.updateLoyaltyCardBalance(
+                    Voucherify.getInstance().getLoyaltyCampaign().getVoucherIds().get(0),
+                    loyaltiesMembersBalanceUpdateRequestBody);
+
+            assertNotNull(responseBody);
+        } catch (ApiException | JsonSyntaxException e) {
+            fail();
+        }
+    }
+
+    @Test
+    public void updateLoyaltyCardBalance2Test() {
+        try {
+            LoyaltiesMembersBalanceUpdateRequestBody loyaltiesMembersBalanceUpdateRequestBody = new LoyaltiesMembersBalanceUpdateRequestBody();
+            loyaltiesMembersBalanceUpdateRequestBody.setPoints(1000);
+
+            LoyaltiesMembersBalanceUpdateResponseBody responseBody = loyalties.updateLoyaltyCardBalance1(
+                    Voucherify.getInstance().getLoyaltyCampaign().getId(),
+                    Voucherify.getInstance().getLoyaltyCampaign().getVoucherIds().get(0),
+                    loyaltiesMembersBalanceUpdateRequestBody);
+
+            assertNotNull(responseBody);
+        } catch (ApiException | JsonSyntaxException e) {
+            fail();
+        }
+    }
+
+    @Test
+    public void loyaltiesMembersTransactionsListTest() {
+        try {
+            LoyaltiesMembersTransactionsListResponseBody responseBody = loyalties.listLoyaltyCardTransactions(
+                    Voucherify.getInstance().getLoyaltyCampaign().getVoucherIds().get(0),
+                    10,
+                    1);
+
+            assertNotNull(responseBody);
+        } catch (ApiException | JsonSyntaxException e) {
+            System.out.println(e);
+            fail();
+        }
+    }
+
+    @Test
+    public void createInBulkLoyaltyTiersTest() {
+        try {
+            List<LoyaltiesTiersCreateInBulkRequestBodyItem> tiers = new ArrayList<>();
+
+            LoyaltyTierBasePoints loyaltyTierBasePoints = new LoyaltyTierBasePoints();
+            loyaltyTierBasePoints.setFrom(0);
+            loyaltyTierBasePoints.setTo(1000);
+
+            LoyaltiesTiersCreateInBulkRequestBodyItem item = new LoyaltiesTiersCreateInBulkRequestBodyItem();
+            item.setName("Bronze");
+            item.setPoints(loyaltyTierBasePoints);
+
+            // UNCOMMENT WHEN YOU CREATED REWARD MANUALLY AND WANT TO TEST AND PROVIDE FIXED
```

Comment:
To mój kod chyba wcześniejszy. Ale imo to do wyrzucenia. Chyba, że można w aktualnej wersji SDK dodać ten reward i da się to przetestować.